### DISCUSSION TITLE
[SOLVED] Plans to support Newsblur?

### DISCUSSION THREAD

**Discussion by pivic** [martinrotter/rssguard#369] [Ideas]
Hi there! Are there any plans to support [Newsblur](https://newsblur.com/)? I'd love that. Cheers!

**Comment by martinrotter** [comment]
Hello @pivic!
Newsblur is nice service with good API. It could certainly be added to RSS Guard via [service plugin](https://github.com/martinrotter/rssguard/blob/master/resources/docs/Feed-formats.md#plugin-api) - it would be probably relatively small plugin with only some hundreds of C++ code - easy to write for any competent programmer.

Unfortunately, I myself have decided that current set of supported services is personally enough for me, and there are other important tasks ahead, so if you would like to have Newsblur in RSS Guard you have some options:
* write the plugin yourself if your are programmer,
* tell some friend of yours, who is a programmer, to write and contribute the plugin,
* or I can write the plugin for you for a monetary donations.

Sorry for late, sincere and direct answer. :)

**Reply by pivic** [reply]
Hi, @martinrotter! Thanks for your brilliant response! I'll take it in and react if I do.
